config/application.rb

```diff
@@ -287,6 +287,9 @@ def try_config_for(*args)
         WARNING
       end
     end
+
+    # Fixes 'DEPRECATION WARNING: Time columns will become time zone aware in Rails 5.1' keeping backwards compatibility
+    config.active_record.time_zone_aware_types = [:datetime]
```

Comment:
Maybe we should not keep compatibility. Does it harm if we don't and have timezone aware?
I agree we can test this later

Reply:
I also thought about it but I didn't investigate earlier. The most informative resource I found is

https://liefery-it-legacy.github.io/blog/2017/10/25/times-in-rails-5.html

From what I could understand it depends on how `Time` columns are used.

I can only find `time` column used in MySQL and PostgreSQL `schema.rb` under `table "line_items"`.  For oracle that column is `date` which is somehow funny.

 I guess we need to test how column is used to understand whether it might be a problem or not.